CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## [2.0.0](https://github.com/OctopusDeploy/push-package-action/compare/v1.2.1...v2.0.0) (2022-07-13)
+
+
+### ⚠ BREAKING CHANGES
+
+* remove unused inputs
+
+### Features
+
+* release 2.0.0 ([#248](https://github.com/OctopusDeploy/push-package-action/issues/248)) ([ba24ea6](https://github.com/OctopusDeploy/push-package-action/commit/ba24ea6be13e5e89c2e0faa4f3d29f7fade95ed1))
```

Comment:
Is "release 2.0.0" a feature?

Reply:
Not really, but for release-please to create the PR it needs a `fix|feat|refactor` in the commit message...